h2o-genmodel/src/test/java/hex/genmodel/easy/EasyPredictModelWrapperWithTargetEncodingTest.java

```diff
@@ -0,0 +1,94 @@
+package hex.genmodel.easy;
+
+import hex.ModelCategory;
+import hex.genmodel.algos.targetencoder.EncodingMap;
+import hex.genmodel.algos.targetencoder.EncodingMaps;
+import hex.genmodel.algos.targetencoder.TargetEncoderMojoModel;
+import hex.genmodel.easy.exception.PredictException;
+import hex.genmodel.easy.prediction.TargetEncoderPrediction;
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static hex.genmodel.utils.SerializationTestHelper.deserialize;
+import static hex.genmodel.utils.SerializationTestHelper.serialize;
+import static org.junit.Assert.*;
+
+public class EasyPredictModelWrapperWithTargetEncodingTest {
```

Comment:
does this test cover the case with numfeatures that @mn-mikke mentioned?

Reply:
@honzasterba As I explained in slack https://h2oai.slack.com/archives/GL84J2LF3/p1563205625015100 @mn-mikke's use case should not be covered in my test (scope of PR) as the goal is only to produce valid encodings and that is it. I'm checking that encodings from Mojo model are the same as from TargetEncoder itself. If Marek was only referring to issue with transformation with TargetEncoderMojoModel then there might be an issue  but I need to see reproducible failing test from Marek as I have covered with test maybe specific case and Marek configured it in a different way.

Reply:
since the sole purpose of this PR is to support SW use-case I think we should have a test that proves it works

Reply:
@honzasterba  Goal of this PR is to add ability to transform with mojo. There is a test that proves that it works. We just have different assumption .... I assumed that number of features should be the same as during the training, but in SW they assume that during transformation they can have only column that are about to be encoded.

Reply:
yes, exactly, please add a test that tests for the SW use-case